cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -240,6 +240,22 @@ TEST(RayClusterModeTest, ExceptionTest) {
   EXPECT_THROW(object1.Get(), ray::internal::RayTaskException);
 }
 
+TEST(RayClusterModeTest, GetActorTest) {
+  ray::ActorHandle<Counter> actor = ray::Actor(RAY_FUNC(Counter::FactoryCreate))
+                                        .SetMaxRestarts(1)
+                                        .SetName("named_actor")
+                                        .Remote();
+  auto named_actor_obj = actor.Task(&Counter::Plus1).Remote();
+  EXPECT_EQ(1, *named_actor_obj.Get());
+
+  auto named_actor_handle_optional = ray::GetActor<Counter>("named_actor");
+  EXPECT_TRUE(named_actor_handle_optional);
+  auto &named_actor_handle = *named_actor_handle_optional;
+  auto named_actor_obj1 = named_actor_handle.Task(&Counter::Plus1).Remote();
+  EXPECT_EQ(2, *named_actor_obj1.Get());
+  EXPECT_FALSE(ray::GetActor<Counter>("not_exist_actor"));
+}
```

Comment:
Why write this test case twice?

Reply:
One is for local mode testing, another one is for cluster mode testing.

Reply:
Why not write the test once and add a switch flag to test it in both local and cluster mode?

Reply:
Our current test cases are divided into two parts:
Local mode test: includes api_test.cc, ray_remote_test.cc
Cluster mode test: includes cluster_mode_test.cc
They have different singleton runtime after init, can't support both local mode and runtime mode at the same time now.